### ISSUE TITLE
ARFF vs Sparse ARFF

### ISSUE THREAD

**Issue by janvanrijn** [openml/OpenML#808]
Is it guaranteed that a sparse arff file is annotated as such? If not, we should take measures to do so.

**Comment by janvanrijn** [comment]
This is important for the to csv functionality for example (in order to show an appropriate message )

**Comment by joaquinvanschoren** [comment]
No, there is no guarantee. Can this be verified in the php check?

**Comment by janvanrijn** [comment]
This probably requires an extension of the php arff checker
